src/tsconfig.json

```diff
@@ -4,9 +4,7 @@
 		"rootDir": "./",
 		"outDir": "../dist",
 		"module": "ESNext",
-		"target": "ESNext",
-		"composite": true,
-		"preserveConstEnums": true
+		"target": "ESNext"
```

Comment:
Use NodeNext / Node16

Reply:
Aside from the fact that that is a module option and not a target option, it then outputs CJS code, not ESM:

![image](https://user-images.githubusercontent.com/4019718/225042163-ca3cc921-53bc-40d2-b71d-e8452addf90d.png)


With current config:

![CleanShot 2023-03-14 at 15 58 12](https://user-images.githubusercontent.com/4019718/225042229-a618a5c5-e047-41f7-ab44-d7f36c237291.png)